aries-site/src/examples/components/page/PageFullExample.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Footer,
+  Grid,
+  Header,
+  List,
+  Page,
+  PageContent,
+  Text,
+} from 'grommet';
+import { Bookmark } from 'grommet-icons';
+
+import serverhealth from '../../../data/mockData/serverhealth.json';
+import { PageHeader, StatusIndicator } from '../../templates';
+import { demoStyle } from './demoStyle';
+
+export const PageFullExample = () => (
+  <Page kind="full" flex="grow" {...demoStyle}>
```

Comment:
Seems like we had to add `flex="grow"` on all of these -- do you feel like that should be a default prop on the component?

Reply:
Or is it more specific to our presentation container?